### PR TITLE
Add exec sub-command

### PR DESCRIPTION
**Pull request by omidasadpour**
/close #470 

Example: 

```shell
sunodo exec /bin/bash
```

image

### CODE REVIEW

**Comment by omidasadpour** [comment]
> Take a look [here](https://oclif.io/docs/args) on how to handle var args, and not require a `-c` option

Fixed . 
Just I don't know how to allow multiple part commands. like `ls -la`

I have to set it like this : 

```shell
sunodo create "ls -la"
```

File: apps/cli/src/commands/exec.ts [hunk 1]
```diff
@@ -7,32 +7,31 @@ import path from "path";
...
 
-    static examples = ["<%= config.bin %> <%= command.id %>"];
+    static examples = ["<%= config.bin %> <%= command.id %> /bin/bash"];
 
     static args = {
         image: Args.string({
             description: "image ID|name",
             required: false,
         }),
+        command: Args.string({
```
Comment (tuler) [inline apps/cli/src/commands/exec.ts:17]: What is the image arg above

**Comment by tuler** [comment]
> I just don't know how to allow multiple part commands. like `ls -la`

Did you try the `static strict = false`, as described in the doc?

**Comment by omidasadpour** [comment]
> Did you try the `static strict = false`, as described in the doc?

Yup, Did not work. 
I'll test again

File: apps/cli/src/commands/exec.ts (hunk 1 above)
Comment (omidasadpour) [reply apps/cli/src/commands/exec.ts:17]: I think we can set a custom image name with it

**Comment by omidasadpour** [comment]
> > Did you try the `static strict = false`, as described in the doc?
> 
> Yup, Did not work. I'll test again

not working

**Comment by omidasadpour** [comment]
> > Take a look [here](https://oclif.io/docs/args) on how to handle var args, and not require a `-c` option
> 
> Fixed . I just don't know how to allow multiple part commands. like `ls -la`
> 
> I have to set it like this :
> 
> ```shell
> sunodo exec "ls -la"
> ```

Actually it does not work because of `-` in. For example I can run this command : 

```
sunodo exec echo $PWD
```

image

**Comment by omidasadpour** [comment]
> > Take a look [here](https://oclif.io/docs/args) on how to handle var args, and not require a `-c` option
> 
> Fixed . I just don't know how to allow multiple part commands. like `ls -la`
> 
> I have to set it like this :
> 
> ```shell
> sunodo exec "ls -la"
> ```

Actually it does not work because of `-`. For example I can run this command without double quotes: 

```shell
sunodo exec ls /home
```

**Comment by tuler** [comment]
Rediscuss the solution for #470 , or if it's necessary

**Pull request closed without merging**